sys/fuchsia/fidl_mappings.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2017 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package fuchsia
```

Comment:
This does not belong to sys/fuchsia package. This package is for system call descriptions rather then metadata. Nobody will look for something else here, and importing it looks weird, none of the other sys/OS packages are imported anywhere (I hope). Please move this to sys/fuchsia/layers or something.

Reply:
Moved to sys/fuchsia/layout since we may have more build layout related details over time.